client/src/components/leftPanel/LeftPanel.js

```diff
@@ -86,10 +108,20 @@ class LeftPanel extends Component {
 		          onChange={this.onShowArchivedChange}
 	          />
           )}
+          {filteredTags.map((tag) => (isTagEditOpen[tag.tag] && <TagEdit
+            key={tag.tag}
+            tag={tag}
+            onClose={this.onCloseTagEdit}
+            isArchived={tag.isArchived}
+            asanaLink={tag.asanaLink}
+            portfolioOptions={portfolioOptions}
+            loadReports={loadReports}
+          />))}
```

Comment:
This does not seem right 🤔. We can edit only one tag at time and thus we should store in state only one tag - the currently edited one (or null if none is being edited).